types/wallet.d.ts

```diff
@@ -7,17 +7,24 @@ import { KeyStore } from './key_store';
 import { Rpc } from './rpc';
 import { TransferParams } from './token';
 import { Utxo } from './utxo';
+export interface BroadcastOptions {
+    headers?: {
+        [key: string]: string;
+    };
+    params?: any[];
+}
+export declare type TransferOptions = BroadcastOptions;
```

Comment:
Is this additional parameter only called broadcast?
For example, the name BroadcastOptions may not be appropriate if you also need to call it when communicating with listUnspent.

Reply:
Yes. we need it only when broadcasting (as for now).